int16 titikX
int16 titikY
int16 target_sudut
int16 kecepatan_x
int16 kecepatan_y
int16 kecepatan_theta
